### PR TITLE
Tophs multiple event request

### PR DESCRIPTION
**Pull request by ccelestial**
Add ability to put multiple events

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ This gem provide a simple way to report analytics events to Amazon Mobile Analyt
 Install the latest stable release:
 
 ```
-$ gem install awsma_rails https://github.com/thumzap/awsma_rails.git
+$ gem install awsma_rails https://github.com/teamapp/awsma_rails.git
```
Comment (pelegb) [inline README.md]: Can't accept pull request with change in installation instructions

File: README.md [hunk 2]
```diff
@@ -7,13 +7,13 @@ This gem provide a simple way to report analytics events to Amazon Mobile Analyt
...
 ```
 
 In Rails, add it to your Gemfile:
 
 ```ruby
-gem 'awsma_rails', :git => 'https://github.com/thumzap/awsma_rails.git'
+gem 'awsma_rails', :git => 'https://github.com/teamapp/awsma_rails.git'
```
Comment (pelegb) [inline README.md]: Can't accept pull request with change in installation instructions

File: lib/awsma_rails/reporter.rb [hunk 3]
```diff
@@ -43,6 +58,29 @@ def report_event(client_id, session_id, app_title, app_package_name, event_name,
 
     private
 
+    def create_analytics_batch_data(events)
+      aws_analytics_data = events.inject([]) do |hash_events, event|
+        event = event.symbolize_keys
```
Comment (pelegb) [inline lib/awsma_rails/reporter.rb]: Combine with create_analytics_data instead of implementing another event to hash function

**Comment by ccelestial** [comment]
Hi @pelegb,

I accidentally created this PR from my team's fork. But, since there are just small changes that you request here. I'll do another PR from my own fork. :)

**Comment by ccelestial** [comment]
Will create a more decent pull-request for this feature.

**Pull request closed without merging**